internal/pkg/template/templates/addons/aurora/env/addons.parameters.yml

```diff
@@ -1,9 +1,9 @@
 Parameters:
   VPCID: 
-    - !If ImportedVPC
+    - !If IsVPCImported
     - !Ref ImportedVPCID
```

Comment:
qq: Previously, from where do we pull the value of this `ImportedVPCID` param?

Reply:
Here: https://github.com/aws/copilot-cli/blob/710bc41fb8a0a8ad1bab8986d8e75c8b8557c617/internal/pkg/template/templates/addons/aurora/env/addons.parameters.yml

It is not actually used in any recent release because the template isn't used yet.